wiz-io/info.json

```diff
@@ -0,0 +1,216 @@
+{
+  "name": "wiz-io",
+  "version": "1.0.0",
+  "label": "Wiz.io",
+  "description": "Wiz provides a comprehensive analysis engine that integrates: Cloud Security Posture Management (CSPM) Kubernetes Security Posture Management (KSPM) Cloud Workload Protection (CWPP) + vulnerability management. Infrastructure-as-Code (IaC) scanning.",
+  "publisher": "Julian Petersohn",
+  "icon_small_name": "download.png",
+  "icon_large_name": "download.png",
+  "category": "Asset Management",
+  "configuration": {
+    "fields": [
+      {
+        "title": "API Endpoint URL",
+        "type": "url",
+        "name": "aPIEndpointURL",
+        "required": true,
+        "visible": true,
+        "editable": true,
+        "value": "https://api.us1.app.wiz.io",
+        "tooltip": "Provide the API endpoint URL for the GraphQL API. The format is as follow: https://api.<location>.app.wiz.io\nThe \"location\" value and the URL can be retrieved from the WIZ tenant settings."
+      },
+      {
+        "title": "Client ID",
+        "type": "text",
+        "name": "clientID",
+        "required": true,
+        "visible": true,
+        "editable": true,
+        "value": "",
+        "tooltip": "Provide the API Client ID generated in the service account of the WIZ deployment."
+      },
+      {
+        "title": "Client Secret",
+        "type": "password",
+        "name": "clientSecret",
+        "required": true,
+        "visible": true,
+        "editable": true,
+        "value": null,
+        "tooltip": "Provide the API Client Secret generated in the service account of the WIZ deployment."
+      },
+      {
+        "title": "Authentication Endpoint URL",
+        "type": "url",
+        "name": "authenticationEndpointURL",
+        "required": false,
+        "visible": true,
+        "editable": true,
+        "value": "https://auth.app.wiz.io/oauth/token",
+        "tooltip": "URL wich is used to retrieve OAuth Token from. The URL can be found at the Service Account page."
```

Comment:
Spelling mistakes of 'which' also complete the sentence by specifying the source.